components/tabs/tab.js

```diff
@@ -30,16 +40,15 @@ class Tab extends TabMixin(LitElement) {
 
 	renderContent() {
 		const contentClasses = {
-			'd2l-tab-handler': true,
 			'd2l-tab-text': true,
 		};
 
 		return html`
 			<div class="${classMap(contentClasses)}">
-				<slot></slot>
+				${this.text}
```

Comment:
This case is to replace our existing default tab behaviour.

Reply:
My initial reaction is to lock this down as well. We might want to limit the content of any slots we provide to ensure consumers don't use things that would break keyboard interaction etc. If you haven't already, we might want to have some discussion with Jeff about the direction we want to go API-wise.